src/index.js

```diff
@@ -49,6 +50,16 @@ function tileReduce(options) {
   return ee;
 }
 
-function bboxTiles(bbox, zoom) {
-  return tilecover.tiles(bboxPolygon(bbox).geometry, {min_zoom: zoom, max_zoom: zoom});
+function parseToTiles(options, zoom) {
+  var poly;
+
+  if (options.tiles) return options.tiles;
+
+  if (options.bbox) {
+    poly = bboxPolygon(options.bbox);
+  } else if (options.geojson) {
+    poly = options.geojson;
+  }
+
+  return tilecover.tiles(normalize(poly).features[0].geometry, {min_zoom: zoom, max_zoom: zoom});
```

Comment:
> The geojson property is also kind of ambiguous — e.g. from an example code it's not obvious that this means GeoJSON polygon that encloses the processing area, but I don't know what would be a better name. area?

Since we're introducing a *type* of option, maybe we should group up the possible values for this option in a top level key to help remove ambiguity: 

```
tilereduce({
  area: { polygon: /* some polygon */}
  // or
  area: { bbox: [/* W,S,E,N */]}
  // or 
  area: { tiles: [/* list of tiles *]}
```



Then the purpose of the argument is very clear.